spec/controllers/news_item_controller_spec.rb

```diff
@@ -1,25 +1,24 @@
 require 'rails_helper'
 
 RSpec.describe NewsItemsController, type: :controller do
+  context 'when accessing a published news_item' do
+    let(:published_news_item) { create :news_item, state: 'published' }
 
-  context "when accessing a published news_item" do
-    let(:published_news_item) { create :news_item, state: "published" }
-
-    it "returns a 200 status code" do
+    it 'returns a 200 status code' do
       get :show, id: published_news_item
       expect(response).to have_http_status(200)
     end
   end
 
-  context "when accessing a non-published news_item" do
-    let(:draft_news_item) { create :news_item, state: "draft" }
-    let(:archived_news_item) { create :news_item, state: "draft" }
+  context 'when accessing a non-published news_item' do
+    let(:draft_news_item) { create :news_item, state: 'draft' }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.